app/graphql/types/integrations/anrok_objects/breakdown_object.rb

```diff
@@ -14,6 +17,14 @@ class BreakdownObject < Types::BaseObject
         def rate
           BigDecimal(object.rate)
         end
+
+        def tax_code
+          object.name&.parameterize(separator: '_')
```

Comment:
would you consider memoize the tax_code, as it is currently calculated twice at `enumed_tax_code` method?

```suggestion
          @tax_code ||= object.name&.parameterize(separator: '_')
```

Reply:
ah, right, thank you! 🙈